integrations/extensions/starter-kits/language-model-conversational-search/README.md

```diff
@@ -177,8 +181,70 @@ If you want to experiment with different prompts, do the following:
+| Search                         | Connects to Watson Discovery to search for documents related to the user query. The "No Action Matches" action has been configured to send all input to this action, so whatever the user enters will be used as the search input. It invokes the "Generate Answer" action to generate a response to the query. |
+| Generate Answer                | Configures the query prompt and document passages resulting from search, and calls the action "Invoke GPT Chat Completion API". It is not meant to be invoked directly, but rather by the "Search" action.                                                                                                      |
+| Invoke GPT Chat Completion API | Connects to OpenAI and, using as context the documents resulting from the search, asks the language model to generate an answer to the user query. It is not meant to be invoked directly, but rather by the "Generate Answer" action.                                                                          |
+
+To use the sample actions:
+
+1. **After having configured both extensions**, download the sample actions from this starter kit: [`discovery-openai-actions.json`](./discovery-openai-actions.json).
+
+1. Use **Actions Global Settings** to upload the JSON file to your assistant. For more information, see [Uploading](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-admin-backup-restore#backup-restore-import).
+
+1. Under "Variables"/"Created by you" (within the Actions page), set the `discovery_project_id` session variable using the project ID value you obtained [when configuring Watson Discovery above](#configure-watson-discovery-extension).
+
+**NOTE**: If you import the actions _before_ configuring the extensions, you will see some errors on the actions because it could not find the extensions. Simply configure the extensions as described above and re-import the action JSON file.
+
+#### Session variables
+
+These are the session variables used in this example. Several of them are automatically set in the sample [discovery-openai-actions.json](discovery-openai-actions.json). You will need to set the `discovery_project_id` to point to the project id for your Watson Discovery collection.
+
+- `discovery_date_version` - Discovery date versions are documented in the [release notes](https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-release-notes).
+- `discovery_project_id`: You **MUST** set this value to be the project ID obtained [when configuring Watson Discovery above](#configure-watson-discovery-extension).
+- `messages` : Input to the OpenAI model; includes the `search_results` and the `model_prompt`.
```

Comment:
This is a bit confusing. Can you add some example here?

Reply:
Should we set the initial value here for that in-session variable?

Reply:
I am not sure what you mean by this `Should we set the initial value here for that in-session variable?`. Could you please clarify?

Reply:
Oh I mean in the session variable like how we set project_id